its/src/test/java/com/sonar/it/scanner/SonarScannerTestSuite.java

```diff
@@ -37,12 +37,12 @@
 
   @ClassRule
   public static final Orchestrator ORCHESTRATOR = Orchestrator.builderEnv()
-    .setSonarVersion(TestUtils.replaceLtsVersion(System.getProperty("sonar.runtimeVersion", "LATEST_RELEASE")))
+    .setSonarVersion(TestUtils.replaceLtsVersion(System.getProperty("sonar.runtimeVersion", "LATEST_RELEASE[6.7]")))
```

Comment:
Should the default be the current LTS?

Reply:
That was not meant to be committed i think, my bad !